packages/slate-theme/src/scripts/sections/product.js

```diff
@@ -127,6 +122,7 @@ theme.Product = (function() {
     updateProductImage: function(evt) {
       var variant = evt.variant;
       var sizedImgUrl = slate.Image.getSizedImageUrl(variant.featured_image.src, this.settings.imageSize);
+      var $image = $(selectors.productFeaturedImage, this.$container);
```

Comment:
If we go with the instance variable suggestion above, we can omit this altogether.